laa_crime_application_store_app/routers/v1/application.py

```diff
@@ -28,6 +32,28 @@
 }
 
 
+@router.get("/v1/applications", response_model=Wrapper)
+async def get_applications(since: int | None = None, db: Session = Depends(get_db)):
+    logger.info("GETTING_APPLICATIONS")
+    applications = (
+        db.query(Application)
+        .filter(Application.updated_at > datetime.fromtimestamp(since or 0))
+        .limit(20)
```

Comment:
Do we want to limit here? Should this be a parameter we pass in?

Reply:
I put a limit here to avoid breaking anything in case it goes wrong, as we plan to run this every 10minutes I would expect 1 or 2 at most to have been updated...
happy to make it a param with a default?